packages/perseus/src/widgets/interactive-graphs/interactive-graph.test.tsx

```diff
@@ -1046,100 +1046,186 @@ describe("Interactive Graph", function () {
+                        directionalAxis: "y",
+                    }),
+                    blankOptions,
+                );
 
-            // Assert
-            expect(point).toHaveAttribute("aria-label", "Function A");
-        });
+                // Assert
+                expect(PlotOfXMock).toHaveBeenCalledTimes(0);
+                expect(PlotOfYMock).toHaveBeenCalledTimes(1);
+            });
 
-        it("should render locked function without aria label by default", () => {
-            // Arrange
-            const simpleLockedFunctionquestion =
-                interactiveGraphQuestionBuilder()
-                    .addLockedFunction("x^2")
-                    .build();
-            const {container} = renderQuestion(
-                simpleLockedFunctionquestion,
-                blankOptions,
-            );
+            it("plots the equation with any supplied domain", () => {
+                // Arrange
+                const PlotOfXMock = jest
+                    .spyOn(Plot, "OfX")
+                    .mockReturnValue(<div>OfX</div>);
+                const expectedParameters = {
+                    color: "#3B3D45",
+                    domain: [-2, 3],
+                    style: "solid",
+                };
 
-            // Act
-            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
-            const point = container.querySelector(".locked-function");
+                // Act
+                renderQuestion(
+                    segmentWithLockedFunction("x^2", {domain: [-2, 3]}),
+                    blankOptions,
+                );
 
-            // Assert
-            expect(point).not.toHaveAttribute("aria-label");
+                // Assert
+                expect(PlotOfXMock).toHaveBeenCalledTimes(1);
+                expect(PlotOfXMock).toHaveBeenCalledWith(
+                    expect.objectContaining(expectedParameters),
+                    {},
+                );
+            });
+
+            it("plots the equation with partially supplied domains", () => {
+                // Arrange
+                const PlotOfXMock = jest
+                    .spyOn(Plot, "OfX")
+                    .mockReturnValue(<div>OfX</div>);
+                const expectedParameters = {
+                    color: "#3B3D45",
+                    style: "solid",
+                };
+
+                // Act - no upper limit specified
+                renderQuestion(
+                    segmentWithLockedFunction("x^2", {domain: [null, 3]}),
+                    blankOptions,
+                );
+
+                // Assert
+                expect(PlotOfXMock).toHaveBeenCalledWith(
+                    expect.objectContaining({
+                        ...expectedParameters,
+                        domain: [-Infinity, 3],
+                    }),
+                    {},
+                );
+
+                // Act - no lower limit specified
```

Comment:
Extending my above nit to all the tests. Can you break this up into two separate tests? A "plots the equation with partially supplied domains (no upper limit specified)" case and a "plots the equation with partially supplied domanis (no lower limit specified)" case.